data/net/minecraft/client/renderer/LevelRenderer.mapping

```diff
@@ -34,8 +34,8 @@ CLASS net/minecraft/client/renderer/LevelRenderer
 		ARG 10 blue
 		ARG 11 alpha
 	METHOD addFrustumQuad (Lcom/mojang/blaze3d/vertex/VertexConsumer;Lorg/joml/Matrix4f;IIIIIII)V
-		ARG 1 consumer
-		ARG 2 matrix
+		ARG 1 buffer
```

Comment:
Why the change from `consumer` to `buffer` in these methods, since the type is `VertexConsumer`?

Reply:
`buffer` is more correct. The `VertexConsumer` is always sourced from `MultiBufferSource.getBuffer()` and represents a buffer.

Reply:
From the perspective of that method, it has no distinction whether the caller gives it a buffer or not. I think it should still be `consumer`, since it is possible something else (a mod, perhaps) can call that method with a `VertexConsumer` that isn't necessarily a buffer from `MultiBufferSource`.